corehq/apps/userreports/tasks.py

```diff
@@ -47,7 +47,7 @@ def _build_indicators(indicator_config_id, relevant_ids):
             # save is a noop if the filter doesn't match
             adapter.save(doc)
             redis_client.srem(redis_key, doc.get('_id'))
-        except DataError as e:
+        except Exception as e:
             logging.exception('problem saving document {} to table. {}'.format(doc['_id'], e))
```

Comment:
Yeah, no other source (they are in couchlog, but just as unlikely to be seen there). I'm mainly viewing this as a way for us to go into the logs and figure out why data is missing from a table, rather than a "something went really wrong" type of thing. The other side of that argument would be that we should strive to never fail with an unknown error in any UCR data source, and therefore should fail louder. I think that would be nice to strive for, though might not be a realistic goal right now (many of our issues are of the form "this thing you said was a number is sometimes a string" and stuff like that). We could turn on emails and just see what happens though if you feel strongly.